--- devkitppc-rules-1.0.0/Makefile	2019-03-10 02:59:56.670994589 +0900
+++ devkitppc-rules-1.0.0/Makefile_	2019-03-10 02:59:45.159073244 +0900
@@ -8,7 +8,7 @@ all:
 	@echo "use dist or install targets"
 
 install:
-	@cp -v $(FILES) $(DESTDIR)/opt/devkitpro/devkitPPC
+	@cp -v $(FILES) $(INSTALLDIR)/devkitPPC
 
 dist:
 	@tar -cJf devkitppc-rules-$(_MAJOR).$(_MINOR).$(_PATCH).tar.xz $(FILES) Makefile
