--- devkitarm-rules-1.0.0/Makefile	2019-02-18 09:58:31.000000000 +0900
+++ devkitarm-rules-1.0.0/Makefile_	2019-03-10 02:48:46.839895635 +0900
@@ -8,8 +8,8 @@ all:
 	@echo "use dist or install targets"
 
 install:
-	@mkdir -p $(DESTDIR)/opt/devkitpro/devkitARM
-	@cp -v $(FILES) $(DESTDIR)/opt/devkitpro/devkitARM
+	@mkdir -p $(INSTALLDIR)/devkitARM
+	@cp -v $(FILES) $(INSTALLDIR)/devkitARM
 
 dist:
 	@tar -cJf devkitarm-rules-$(_MAJOR).$(_MINOR).$(_PATCH).tar.xz $(FILES) Makefile
