バイナリーファイル RG350_buildroot_org/.git/index とRG350_buildroot/.git/index は異なります
diff -uprN RG350_buildroot_org/board/opendingux/gcw0/post-build-script.sh RG350_buildroot/board/opendingux/gcw0/post-build-script.sh
--- RG350_buildroot_org/board/opendingux/gcw0/post-build-script.sh	2020-05-14 18:42:03.848975000 +0900
+++ RG350_buildroot/board/opendingux/gcw0/post-build-script.sh	2020-05-14 18:45:39.144231109 +0900
@@ -6,7 +6,7 @@ TARGET_DIR=$1
 for i in ar as ld ld.bfd nm objcopy ranlib strip ; do
 	rm -f ${TARGET_DIR}/usr/bin/$i
 done
-rm -rf ${TARGET_DIR}/usr/mipsel-gcw0-linux-uclibc
+rm -rf ${TARGET_DIR}/usr/mipsel-rg350-linux-uclibc
 
 # We use xinetd, so no need for a startup script for the SSH daemons
 # and the FTP server.
diff -uprN RG350_buildroot_org/board/opendingux/gcw0/uclibc.config RG350_buildroot/board/opendingux/gcw0/uclibc.config
--- RG350_buildroot_org/board/opendingux/gcw0/uclibc.config	2020-05-14 18:42:03.848975000 +0900
+++ RG350_buildroot/board/opendingux/gcw0/uclibc.config	2020-05-14 18:48:07.115728973 +0900
@@ -61,7 +61,7 @@ UCLIBC_HAS_FPU=y
 DO_C99_MATH=y
 # DO_XSI_MATH is not set
 UCLIBC_HAS_FENV=y
-KERNEL_HEADERS="/home/mth/gcw0/buildroot/output/toolchain/linux/include"
+KERNEL_HEADERS="/home/mth/rg350/buildroot/output/toolchain/linux/include"
 HAVE_DOT_CONFIG=y
 
 #
@@ -240,7 +240,7 @@ UCLIBC_BUILD_NOEXECSTACK=y
 #
 # Development/debugging options
 #
-CROSS_COMPILER_PREFIX="/opt/gcw0-toolchain/usr/bin/mipsel-gcw0-linux-uclibc-"
+CROSS_COMPILER_PREFIX="/opt/rg350-toolchain/usr/bin/mipsel-rg350-linux-uclibc-"
 UCLIBC_EXTRA_CFLAGS=""
 # DODEBUG is not set
 DOSTRIP=y
diff -uprN RG350_buildroot_org/configs/rg350_defconfig RG350_buildroot/configs/rg350_defconfig
--- RG350_buildroot_org/configs/rg350_defconfig	2020-05-14 18:42:03.864975000 +0900
+++ RG350_buildroot/configs/rg350_defconfig	2020-05-14 18:55:53.662118330 +0900
@@ -2,7 +2,7 @@ BR2_mipsel=y
 BR2_mips_xburst=y
 # BR2_MIPS_SOFT_FLOAT is not set
 BR2_OPTIMIZE_2=y
-BR2_TOOLCHAIN_BUILDROOT_VENDOR="gcw0"
+BR2_TOOLCHAIN_BUILDROOT_VENDOR="rg350"
 BR2_KERNEL_HEADERS_3_12=y
 BR2_UCLIBC_CONFIG="board/opendingux/gcw0/uclibc.config"
 BR2_TOOLCHAIN_BUILDROOT_LARGEFILE=y
diff -uprN RG350_buildroot_org/rebuild.sh RG350_buildroot/rebuild.sh
--- RG350_buildroot_org/rebuild.sh	2020-05-14 18:42:03.996975000 +0900
+++ RG350_buildroot/rebuild.sh	2020-05-15 03:31:35.983394210 +0900
@@ -3,15 +3,15 @@ set -e
 
 # Get the latest version.
 echo "Updating local git repository..."
-git pull
+#git pull
 
 # Use the default config and patch it to point to our install location.
-make gcw0_defconfig
-sed -ie 's%^BR2_HOST_DIR=.*$%BR2_HOST_DIR="/opt/gcw0-toolchain"%' .config
+make rg350_defconfig
+sed -ie 's%^BR2_HOST_DIR=.*$%BR2_HOST_DIR="/opt/rg350-toolchain"%' .config
 
 # Clear the install location.
 echo "Clearing install location..."
-rm -rf /opt/gcw0-toolchain/*
+rm -rf /opt/rg350-toolchain/*
 
 # Clear the build location.
 echo "Clearing build location..."
@@ -39,4 +39,4 @@ nice make BR2_EXTERNAL=board/opendingux
 
 # Create packages.
 echo "Creating packages..."
-tar -C/opt --exclude='.fakeroot.*' -jcf opendingux-gcw0-toolchain.`date +'%Y-%m-%d'`.tar.bz2 gcw0-toolchain
+tar -C/opt --exclude='.fakeroot.*' -jcf opendingux-rg350-toolchain.tar.bz2 rg350-toolchain
